nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -164,8 +164,9 @@ let
         proxy_send_timeout      ${cfg.proxyTimeout};
         proxy_read_timeout      ${cfg.proxyTimeout};
         proxy_http_version      1.1;
-        include ${recommendedProxyConfig};
-      ''}
+        # don't let clients close the keep-alive connection to upstream
+        proxy_set_header        "Connection" "";
```

Comment:
Do you have a reference for this recommendation?

Reply:
https://www.nginx.com/blog/avoiding-top-10-nginx-configuration-mistakes/#no-keepalives

Reply:
Thanks for the link. I now understand why this change is recommended. But it changes the existing behaviour of nginx, you should spend an entry in the release notes.

Reply:
Added a release notes entry